packages/client/src/options.ts

```diff
@@ -13,6 +13,11 @@ export interface ConnectionOptions {
    * @defaultValue 10000
    */
   timeout?: number
+  /**
+   * stream timeout for query (grpc timeout)
```

Comment:
Just clarification what is the problem with timeout:

```suggestion
   * stream timeout for query (grpc timeout). The gRPC doesn't apply the timeout to HTTP operations as is defined above. To successfully close a call to the gRPC endpoint, the queryTimeout must be specified. Without this timeout, a gRPC call might end up in an infinite wait state.
```